File: 824_goat_latin.js
var toGoatLatin = function(S) {
    let vowels = 'aeiou';
    let words = S.split(' ')
    let ans = [];
    for(let i = 0; i < words.length; i++){
        let word = words[i];
        if (vowels.includes(word[0])){
            ans.push(word + 'maa' + 'a'.repeat(i))
        } else {
        let before = word[0];
        let after = word.slice(1, word.length);
        let ending = 'maa' + 'a'.repeat(i);
        ans.push(after + before + ending);
        }
        
    }
    
    return ans.join(' ');
};

File: 824_goat_latin.py
def toGoatLatin(self, S: str) -> str:
    amt = 0
    vowels = 'aeiouAEIOU'
    ans = []
    words = S.split(' ')
    for word in words:
        if word[0] in vowels:
            ans.append(word + 'maa' + amt * 'a')
        else:
            ans.append(word[1:] + word[0] + 'maa' + amt * 'a')
        amt += 1
    
    return ' '.join(ans)
        